style.css

```diff
@@ -99,3 +99,413 @@ header nav a {
 .socail-media-icon a {
   margin-right: 18px;
 }
+
+.wraper {
+  margin: 0 auto;
+}
```

Comment:
- [ ] Kindly as specified by the previous reviewer and as part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) since this is the main container for the `works` section you should use a `display: grid` to layout the elements. This will give you the flexibility of aligning your items in a 2-dimensional manner which will come in handy when you scale up to a desktop version. ✔